<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RadioAstronomySensorDialog</class>
 <widget class="QDialog" name="RadioAstronomySensorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>514</width>
    <height>746</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Liberation Sans</family>
    <pointsize>9</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sensor Control Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="sensor1Group">
        <property name="title">
         <string>Sensor 1</string>
        </property>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="3" column="1">
          <widget class="QLineEdit" name="sensor1Device"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="sensor1DeviceLabel">
           <property name="text">
            <string>Device</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="sensor1InitLabel">
           <property name="text">
            <string>Init</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QPlainTextEdit" name="sensor1Init"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="sensor1MeasureLabel">
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="sensor1Measure"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="sensor1NameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="sensor1Name"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="sensor1EnabledLabel">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="sensor1Enabled">
           <property name="toolTip">
            <string>Check to enable measurements from this device</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="sensor2Group">
        <property name="title">
         <string>Sensor 2</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="sensor2DeviceLabel">
           <property name="text">
            <string>Device</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="sensor2InitLabel">
           <property name="text">
            <string>Init</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QPlainTextEdit" name="sensor2Init"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="sensor2EnabledLabel">
           <property name="text">
            <string>Enabled</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="sensor3NameLabel">
           <property name="text">
            <string>Name</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="sensor2MeasureLabel">
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QLineEdit" name="sensor2Device"/>
         </item>
         <item row="2" column="2">
          <widget class="QLineEdit" name="sensor2Name"/>
         </item>
         <item row="5" column="2">
          <widget class="QLineEdit" name="sensor2Measure"/>
         </item>
         <item row="1" column="2">
          <widget class="QCheckBox" name="sensor2Enabled">
           <property name="toolTip">
            <string>Check to enable measurements from this device</string>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="timingGroup">
        <property name="title">
         <string>Timing</string>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="periodLabel">
           <property name="text">
            <string>Measurement period (seconds)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QDoubleSpinBox" name="period">
           <property name="toolTip">
            <string>Delay in seconds between measurements being made</string>
           </property>
           <property name="decimals">
            <number>3</number>
           </property>
           <property name="minimum">
            <double>0.001000000000000</double>
           </property>
           <property name="maximum">
            <double>100.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>period</tabstop>
 </tabstops>
 <resources>
  <include location="../../../sdrgui/resources/res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RadioAstronomySensorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RadioAstronomySensorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
